issue.go

```diff
@@ -78,6 +78,7 @@ type EditIssueOption struct {
 	Body      *string `json:"body"`
 	Assignee  *string `json:"assignee"`
 	Milestone *int64  `json:"milestone"`
+	Status    *string `json:"state"`
```

Comment:
@Unknwon changed the json-element to `state` but kept the struct-member as `Status` since the rest of the Gogs code-base has it named `Status`...